runtime/src/main/kotlin/kotlin/native/Platform.kt

```diff
@@ -81,6 +81,14 @@ public object Platform {
     public val isDebugBinary: Boolean
         get() = Platform_isDebugBinary()
 
+    /**
+     * If the memory leak checker is activated, by default true in debug mode, false in release.
+     * When memory leak checker is activated, and leak detected on process termination process
+     * is terminated with non-zero exit code.
+     */
+    public var hasMemoryLeakChecker: Boolean
```

Comment:
The name doesn't correspond to the behaviour.
Maybe `isMemoryLeakCheckerEnabled`?

Reply:
Renamed.